examples/chart/teleport-kube-agent/values.yaml

```diff
@@ -99,6 +99,67 @@ databases: []
 #      "*": "*"
 databaseResources: []
 
+################################################################
+# Values that must be provided if Windows Desktop access is enabled.
+################################################################
+
+# AD or non-AD configuration
+
+# (optional) Determines whether desktop sessions will show a user-selected wallpaper
+# vs a system-default, single-color wallpaper. For improved visual performance,
+# set this to false (its default value).
+#showWindowsDesktopWallpaper: false
+
+# (optional) ldap contains configuration keys used when connecting Teleport
+# to an Active Directory domain. This enables the discovery service for
+# Windows desktops belonging to an Active Directory domain configured for
+# Teleport access.
+#windowsDesktopLDAP:
+# addr:     '$LDAP_SERVER_ADDRESS'
+# domain:   '$LDAP_DOMAIN_NAME'
+# username: '$LDAP_USERNAME'
+# sid: '$LDAP_USER_SID'
+# insecure_skip_verify: false
+# server_name: '$SERVER_NAME'
+# ldap_ca_cert: |
+#   -----BEGIN CERTIFICATE-----
+#     *certificate data*
+#   -----END CERTIFICATE-----
+
+
+# (optional) hosts is a list of hostnames to register as WindowsDesktop
```

Comment:
I assume you copied this text from somewhere and forgot to rename it. There are 2-3 places in this file where you refer to something with the wrong name.

Reply:
I've gone back through and these look correct.  I took the text from the reference section.